packages/terra-form/src/components/Field.jsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/jsx-boolean-value, jsx-a11y/label-has-for */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  /**
+   * Children the Field contains
+   */
+  children: PropTypes.node,
+  /**
+   * Error message for when the input is invalid
+   */
+  error: PropTypes.node,
+  /**
+   * Help element to display with the input
+   */
+  help: PropTypes.node,
+  /**
+   * Determines whether the field in an inline field
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Label of the input
+   */
+  label: PropTypes.node,
+};
```

Comment:
I believe we will also want to add a prop for `required` to this component. In the prior code base, we used a the presence of a class, `.field-required`, to alter the label display to help indicate that a field was a required input. 

With the required prop, we could add a class `terra-Form-field--required` when it is set to true to alter the styling.
Here is a sample of the css code from the prior code base:

```css
.field-required > label:first-child {
  &:before{
    .padding-end(5px);
    color: @text-error;
    content: '*';
    display: inline-block;
    font-weight: @bold-font-weight;
  }
}
```